src/app/agents/benchmarks/benchmarks.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="d-flex justify-content-between flex-wrap flex-md-nowrap align-items-center py-4">
+    <div class="d-block mb-4 mb-md-0">
+        <h2 class="h4">Benchmarks</h2>
```

Comment:
You can change this block for this template

<!-- Header -->
<app-page-title [title]="Benchmarks'" [subbutton]="false"></app-page-title>